app/views/govuk_publishing_components/components/_list.html.erb

```diff
@@ -20,7 +25,9 @@
 <% if items.any? %>
   <%= content_tag list_tag, class: classes, id: id, "aria-label": aria_label do %>
     <% items.each do |item| %>
-      <li><%= raw(item) %></li>
+      <%= tag.li(id: list_id) do %>
+        <%= raw(item) %>
+      <% end %>
```

Comment:
Am I reading this right - would this put the same `id` on each of the list items in the list?

Reply:
Just checked, and the same `id` is added to different list elements.